java/ql/src/semmle/code/java/dataflow/FlowSources.qll

```diff
@@ -122,6 +123,15 @@ private class SpringMultipartRequestSource extends RemoteFlowSource {
   override string getSourceType() { result = "Spring MultipartRequest getter" }
 }
 
+class PlayParameterSource extends RemoteFlowSource {
+  PlayParameterSource() {
+    exists(PlayActionMethodQueryParameter p | p = this.asParameter()) or
+    exists(PlayMVCHTTPRequestHeaderMethods m | m.getQueryString() = this.asExpr())
```

Comment:
Is this redundant now that `getQueryString` is a `RemoteTaintedMethod`?

Reply:
Yup done, removed.